jwst/pipeline/calwebb_image3.py

```diff
@@ -85,6 +85,13 @@ def process(self, input_data):
                 input_models = self.skymatch(input_models)
                 input_models = self.outlier_detection(input_models)
 
+            elif self.skymatch.skymethod == 'match':
```

Comment:
@hbushouse 

> So the assumption here is that a single input image doesn't appear in the form of a `ModelContainer`

Not necessarily. A single image could appear as a `ModelContainer` but then `has_groups` is going to be `False` because `has_groups = len(input_models.group_names) > 1` defined above this branch. Here I _assume_ a single image cannot belong to multiple groups and I also assume that number of `group_names` is equal to the number of groups.